VoltAir/resources/values-Spanish.properties

```diff
@@ -0,0 +1,16 @@
+Finish=Fin
+Connect controllers to play with your friends!=Conectar controles para jugar con tus amigos!
+No=No
+Yes=Yes
+Quit?=Salir?
+Loading=Cargando
+Resume=Continuar
+Main Menu=Menu Principal
```

Comment:
@diegowald Don't use spaces in property names. Change to `main.menu=Menu principal`